### PR TITLE
[numba] update context to address reference to external lecture series

### PR DESCRIPTION
**Pull request by mmcky**
This PR adjusts a `:doc:` reference to refer to `python-intro` lecture series.

### CODE REVIEW

**Comment by mmcky** [comment]
fixes #1

**Review by najuzilu — COMMENTED** [review]
Nice @mmcky!

File: source/rst/numba.rst [hunk 1]
```diff
@@ -529,7 +529,8 @@ Compare speed with and without Numba when the sample size is large.
 Exercise 2
 ----------
 
-Later we'll learn all about :doc:`finite-state Markov chains <finite_markov>`.
+In the `Introduction to Quantitative Economics with Python <https://python-intro.quantecon.org>`__ lecture series we'll
```
Comment (jstac) [inline source/rst/numba.rst]: Thanks Matt, that's great.  Perhaps "we'll" should be "you can" --- I'm not sure everyone will go on to the next lecture series.

Comment (jstac) [reply source/rst/numba.rst]: Otherwise all good.

Comment (mmcky) [reply source/rst/numba.rst]: thanks @jstac good suggestion.

**Review by najuzilu — COMMENTED** [review]
Great!

**Pull request merged**